### PR TITLE
Fix packaging of the conversion library

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
Small fix in order to export the conversion library properly.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -50,16 +50,21 @@ endif()
 #
 install(
   TARGETS ${PROJECT_NAME}_conversion
-  EXPORT ${TARGETS_EXPORT_NAME}
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+  EXPORT export_${PROJECT_NAME}
```
Comment (nim65s) [inline CMakeLists.txt:53]: Do you have any pointer to documentation for that, or explanations ?
I mean: `TARGETS_EXPORT_NAME` is defined in jrl-cmakemodules, so if this is wrong, it should be updated there

File: CMakeLists.txt [hunk 2]
```diff
@@ -50,16 +50,21 @@ endif()
...
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  RUNTIME DESTINATION bin
```
Comment (nim65s) [inline CMakeLists.txt:56]:
```suggestion
  LIBRARY DESTINATION $[CMAKE_INSTALL_LIBDIR}
  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
```

File: CMakeLists.txt (hunk 1 above)
Comment (MaximilienNaveau) [reply CMakeLists.txt:53]: It was a bad copy-paste from my part. It should be removed. I am not using jrl-cmakemodules but ament here.

Comment (MaximilienNaveau) [reply CMakeLists.txt:53]: Actually the logic is set by ament below line 64:
`ament_export_targets(export_${PROJECT_NAME} HAS_LIBRARY_TARGET)`
Here you define the export name.

**Pull request merged**